# HG changeset patch
# User Tatsuro Matsuoka <tmacchant@yahoo.co.jp>
# Date 1431543101 -7200
#      Wed May 13 20:51:41 2015 +0200
# Branch stable
# Node ID 7e01182ee36c68ac9b6b6820452756f2d6fc990f
# Parent  f185653b2f5ea6c8c98e33ce6eb285fb59b899a6
ja_JP.ts: Update japanese translation (patch #8642)

diff --git a/libgui/languages/ja_JP.ts b/libgui/languages/ja_JP.ts
--- a/libgui/languages/ja_JP.ts
+++ b/libgui/languages/ja_JP.ts
@@ -903,17 +903,17 @@
         <location line="+62"/>
         <location line="+28"/>
         <source>Octave Editor</source>
         <translation>Octave エディタ</translation>
     </message>
     <message>
         <location line="-318"/>
         <source>Octave Files (*.m);;All Files (*)</source>
-        <translation>Octave ファイル;;全てのファイル (*)</translation>
+        <translation>Octave ファイル (*.m);;全てのファイル (*)</translation>
     </message>
     <message>
         <location line="-40"/>
         <source>New Function</source>
         <translation>新しい関数</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -944,17 +944,17 @@ is already open in the editor</source>
     <message>
         <location line="+2"/>
         <source>Save File &amp;As...</source>
         <translation>ファイルを別名で保存 (&amp;A)...</translation>
     </message>
     <message>
         <location line="+14"/>
         <source>Print...</source>
-        <translation type="unfinished">陰圧...</translation>
+        <translation>プリント...</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>&amp;Undo</source>
         <translation>元に戻す (&amp;U) </translation>
     </message>
     <message>
         <location line="+3"/>
@@ -1106,62 +1106,62 @@ is already open in the editor</source>
     <message>
         <location line="+2"/>
         <source>Cut Line</source>
         <translation>行を切り取り</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Delete to Start of Word</source>
-        <translation type="unfinished"></translation>
+        <translation>単語の始まりまで削除</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete to End of Word</source>
-        <translation type="unfinished"></translation>
+        <translation>単語の終わりまで削除</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete to Start of Line</source>
-        <translation type="unfinished"></translation>
+        <translation>行の始まりまで削除</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete to End of Line</source>
-        <translation type="unfinished"></translation>
+        <translation>行末まで削除</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Duplicate Selection/Line</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Transpose Line</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Show Completion List</source>
-        <translation type="unfinished"></translation>
+        <translation>補完リストの表示(&amp;s)</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Format</source>
-        <translation>フォーマット (&amp;)</translation>
+        <translation>フォーマット (&amp;F)</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Uppercase Selection</source>
-        <translation type="unfinished"></translation>
+        <translation>大文字を選択(&amp;U)</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Lowercase Selection</source>
-        <translation type="unfinished"></translation>
+        <translation>小文字を選択(&amp;L)</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>&amp;Indent</source>
         <translation>インデント (&amp;I)</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -1231,17 +1231,17 @@ is already open in the editor</source>
     <message>
         <location line="-120"/>
         <source>Show Line &amp;Endings</source>
         <translation>行末の表示 (&amp;E)</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Show &amp;Indentation Guides</source>
-        <translation type="unfinished"></translation>
+        <translation>インデンテーションのガイドを表示 (&amp;I)</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Show Long Line &amp;Marker</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+6"/>
@@ -2212,32 +2212,32 @@ Click &apos;Next&apos; to create a confi
     <message>
         <location line="+20"/>
         <source>File Browser</source>
         <translation>ファイルブラウザ</translation>
     </message>
     <message>
         <location line="-84"/>
         <source>Step In</source>
-        <translation type="unfinished"></translation>
+        <translation>ステップイン</translation>
     </message>
     <message>
         <location line="-152"/>
         <source>Load Workspace...</source>
         <translation>ワークスペースのロード...</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Save Workspace As...</source>
         <translation>ワークスペースを別名で保存...</translation>
     </message>
     <message>
         <location line="+153"/>
         <source>Step Out</source>
-        <translation type="unfinished"></translation>
+        <translation>ステップアウト</translation>
     </message>
     <message>
         <location line="+94"/>
         <source>Reset Default Window Layout</source>
         <translation>既定のウィンドウのレイアウト</translation>
     </message>
     <message>
         <location line="+16"/>
@@ -2252,17 +2252,17 @@ Click &apos;Next&apos; to create a confi
     <message>
         <location line="+3"/>
         <source>Contribute to Octave</source>
         <translation>Octave への貢献</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Octave Developer Resources</source>
-        <translation type="unfinished">Octave 開発チーム</translation>
+        <translation>Octave 開発者のリソース</translation>
     </message>
     <message>
         <location line="+14"/>
         <source>On Disk</source>
         <translation>ディスク上</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -3025,17 +3025,17 @@ Octave GUI はオープンできませんでした．</translation>
         <location filename="../src/ui-settings-dialog.h" line="-28"/>
         <source>command line (%f=file, %l=line):</source>
         <translation>コマンドライン(%f=ファイル名, %l=行数): </translation>
     </message>
     <message>
         <location line="+489"/>
         <location filename="../src/ui-settings-dialog.h" line="+31"/>
         <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select font, font size (as a difference from the default size), font style (&lt;b&gt;b&lt;/b&gt;old, &lt;b&gt;i&lt;/b&gt;talic, &lt;b&gt;u&lt;/b&gt;nderline), text color, and background color (for the latter, the color magenta (255,0,255) is a placeholder for the default background color).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;フォントの種類, サイズ (既定のサイズからの差), フォントのスタイル (ボールド(&lt;b&gt;b&lt;/b&gt;), イタリック(&lt;b&gt;i&lt;/b&gt;), 下線(&lt;b&gt;u&lt;/b&gt;)), テキストの色, 背景色 (後者について, マジェンダ (255,0,255) は既定の背景色のプレースホルダー).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+32"/>
         <location filename="../src/ui-settings-dialog.h" line="+12"/>
         <source>Terminal</source>
         <translation>端末</translation>
     </message>
     <message>
@@ -3193,35 +3193,35 @@ Octave GUI はオープンできませんでした．</translation>
         <location filename="../src/ui-settings-dialog.h" line="+2"/>
         <source>Restore last directory of previous session</source>
         <translation>一つ前のセッションの最後のディレクトリを保存する</translation>
     </message>
     <message>
         <location line="+106"/>
         <location filename="../src/ui-settings-dialog.h" line="+3"/>
         <source>Colors for variable attributes </source>
-        <translation type="unfinished"></translation>
+        <translation>変数の属性の色</translation>
     </message>
     <message>
         <location line="+34"/>
         <location filename="../src/ui-settings-dialog.h" line="+22"/>
         <source>Shortcuts</source>
         <translation>ショートカット</translation>
     </message>
     <message>
         <location line="+31"/>
         <location filename="../src/ui-settings-dialog.h" line="-18"/>
         <source>Select this option to prevent conflicts with readline shortcuts</source>
-        <translation type="unfinished"></translation>
+        <translation>Realineのショートカットとのコンフリクトを防ぐためにこのオプションを選択する</translation>
     </message>
     <message>
         <location line="+3"/>
         <location filename="../src/ui-settings-dialog.h" line="+2"/>
         <source>Disable global shortcuts when terminal window has focus</source>
-        <translation type="unfinished"></translation>
+        <translation>端末のウィンドウがフォーカスされている時グローバルなショートカットを無効にする</translation>
     </message>
     <message>
         <location line="+31"/>
         <location line="+28"/>
         <location filename="../src/ui-settings-dialog.h" line="+1"/>
         <location line="+4"/>
         <source>Use this set</source>
         <translation>このセットを使用する</translation>
@@ -3259,29 +3259,29 @@ Octave GUI はオープンできませんでした．</translation>
         <location filename="../src/ui-settings-dialog.h" line="+7"/>
         <source>Action</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+10"/>
         <location filename="../src/ui-settings-dialog.h" line="-1"/>
         <source>Set 1 Default</source>
-        <translation type="unfinished"></translation>
+        <translation>セット 1 を既定にする</translation>
     </message>
     <message>
         <location line="+5"/>
         <location filename="../src/ui-settings-dialog.h" line="-1"/>
         <source>Set 1 Actual</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+5"/>
         <location filename="../src/ui-settings-dialog.h" line="-1"/>
         <source>Set 2 Default</source>
-        <translation type="unfinished"></translation>
+        <translation>セット 2 を既定にする</translation>
     </message>
     <message>
         <location line="+10"/>
         <location filename="../src/ui-settings-dialog.h" line="-1"/>
         <source>Set 2 Actual</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
@@ -3697,32 +3697,32 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+3"/>
         <source>Find and Replace</source>
         <translation>検索と置換</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete to Start of Word</source>
-        <translation type="unfinished"></translation>
+        <translation>単語の始まりまで削除</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete to End of Word</source>
-        <translation type="unfinished"></translation>
+        <translation>単語の終わりまで削除</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete to Start of Line</source>
-        <translation type="unfinished"></translation>
+        <translation>行の始まりまで削除</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete to End of Line</source>
-        <translation type="unfinished"></translation>
+        <translation>行末まで削除</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete Line</source>
         <translation>行削除</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -3752,22 +3752,22 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+2"/>
         <source>Uncomment Selection</source>
         <translation>選択部をコメントでなくす</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Uppercase Selection</source>
-        <translation type="unfinished"></translation>
+        <translation>選択部を大文字にする</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Lowercase Selection</source>
-        <translation type="unfinished"></translation>
+        <translation>選択部を小文字にする</translation>
     </message>
     <message>
         <location line="+4"/>
         <location line="+5"/>
         <source>Indent Selection</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
